QgisModelBaker/libqgsprojectgen/dbconnector/config.py

```diff
@@ -0,0 +1,73 @@
+IGNORED_SCHEMAS = [
```

Comment:
```suggestion
IGNORED_PG_SCHEMAS = [
```

Reply:
They are used in the MSSQL implementations as well. I guess they have the same names, no?

Reply:
pg_catalog is postgres specific, information_schema maybe not. I'd add them to both.